inline-translation/lib/index.js

```diff
@@ -222,6 +222,8 @@ registerPopoverHandlers = function() {
 					translationPair.updateAllTranslations( data[ originalId ], currentUserId );
 					makeTranslatable( translationPair, $node );
 					notifyTranslated( translationPair );
+
+					jQuery( '.translator-translatable.translator-untranslated' ).webuiPopover( 'show' );
```

Comment:
Something like this could find the visually next node.
```suggestion
					var untranslated = jQuery( '.translator-untranslated' );
					untranslated.eq( untranslated.index( $node ) + 1 ).webuiPopover( 'show' );
```

Reply:
I just added `:visible` to the selector and it now jumps to next visible element.